pkg/api/dbaas/mongodbatlasinstance_types.go

```diff
@@ -19,6 +19,18 @@ import (
 	dbaasv1alpha1 "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
 )
 
+const (
```

Comment:
Since those constants are defined for all providers, should they be defined in github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1 and used directly?

Reply:
It is a good idea. I'll update the dbaas PR to add the constants.
But I think we should update the provider operators with the constants in another PR after the dbaas PR is merged. so that the provider operators' `go.mod` could download the latest dbaas api.
In the meantime, we should merge this PR with the dbaas PR, otherwise the provider operators will not work.

Reply:
Yes the dbaas oeperator PR should be merged first.

Reply:
I've added the constants in the DBaaS PR, will update the provider operators once the DBaaS PR is merged.